PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- Open this PR as draft while it is not ready -->
+
+### Description
+
+<!-- Describe your solution for the issue here. Adding videos, screenshots and expected logs may help the others to understand what is new.
+-->
+
+### Related Issues
+<!--
+Use keywords like 'close' or 'solves' to link this PR to an issue.
+For example:
+
+Closes #12345
+Unblocks #54321
+-->
+
+### Progress
+
+<!-- If your PR is WIP, use checkboxes to show that you did and what you have to do. For example: -->
+
+- [x] New endpoint created;
+- [ ] Update organizations;
+- [ ] Create tests;
+
+This PR solves #12345
```

Comment:
I guess this shouldn't be here (handled at `Related Issues` section)

Reply:
Moved to `Related Issues` section